pw_mgr/password_manager_test.py

```diff
@@ -1,17 +1,15 @@
 import os
-import sys
+import pathlib
 
-
-# Structure of project needs to be fixed to make this nicer
-sys.path.append('.')
 from password_manager import write_key
 
 def test_write_key():
     """Test that key gets written"""
+    parent_dir = pathlib.Path(__file__).parent
 
-    if os.path.exists("./key.key"):
-        os.remove("./key.key")
+    if os.path.exists(f"{parent_dir}/key.key"):
+        os.remove(f"{parent_dir}/key.key")
```

Comment:
OAKY! This is something I was gonna ask you about. I've been trying to figure out how to apply relative paths into a script so I can define a path that would work on any machine instead of an absolute path (I think that's the appropriate term?). Stackoverflow gave me some really specific solutions depending on if the file I'm trying to target was in a parent folder, sibling folder, or subfolder. This week, I want to work on writing the .key and .txt files into the pw_mgr folder or maybe even its own sibling folder.